backend/src/test/java/de/neuefische/backend/service/MongoUserServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.model.MongoUser;
+import de.neuefische.backend.model.MongoUserRequest;
+import de.neuefische.backend.model.Traveller;
+import de.neuefische.backend.repository.MongoUserRepo;
+import de.neuefische.backend.repository.TravellerRepo;
+import org.junit.jupiter.api.Test;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class MongoUserServiceTest {
+    MongoUserRepo mongoUserRepo = mock(MongoUserRepo.class);
+    TravellerRepo travellerRepo = mock(TravellerRepo.class);
+    UuidGeneratorService uuidGeneratorService = mock(UuidGeneratorService.class);
+    MongoUserService mongoUserService = new MongoUserService(travellerRepo, uuidGeneratorService, mongoUserRepo);
+
+    @Test
+    void addNewUserAndTraveller_expect_givenId() {
```

Comment:
What I really like about this, is how you structured the blocks into _blocks of thought_ :thought_balloon: